src/main/java/nl/s63b/europeanintegration/jms/DomainTranslator.java

```diff
@@ -0,0 +1,99 @@
+package nl.s63b.europeanintegration.jms;
+
+import com.S63B.domain.Entities.Car;
+import com.S63B.domain.Entities.Invoice;
+import com.S63B.domain.Entities.LicensePlate;
+import com.S63B.domain.Entities.Tracker;
+import nl.s63b.europeanintegration.jms.kotlin.Countries;
+import org.joda.time.DateTime;
+
+import java.time.Instant;
+import java.util.Date;
+
+/**
+ * Created by Kevin.
+ */
+public class DomainTranslator {
+    private static DomainTranslator instance;
+
+    private DomainTranslator() {
+    }
+
+    public static DomainTranslator getInstance() {
+        if (instance == null) {
+            instance = new DomainTranslator();
+        }
+        return instance;
+    }
+
+    public static Car jmsToCar(nl.s63b.europeanintegration.jms.kotlin.Car car) {
+        Car newCar = new Car();
+
+        //Plate
+        LicensePlate plate = new LicensePlate();
+        plate.setLicense(car.getLicensePlate());
+        newCar.setLicensePlate(plate);
+        //Tracker
+        Tracker tracker = new Tracker("", car.getCountryOfOrigin().name());
```

Comment:
Helaas, het car object waar meer gewerkt wordt zit alleen het volgende : 
- licenseplate
- country of origin
- country of destination
- boolean stolen